### ISSUE TITLE
JS console is not working

### ISSUE THREAD

**Issue by jacogr** [polkadot-js/apps#1795]
i.e. this doe not execute -

```js
const events = await api.query.system.events(); // .at('0x3db7fb0c4b6a18a29b53bb177d5b897fd636a35397c45867cf2f490bffdde535');
console.log(events)
```

**Comment by jacogr** [comment]
... rather... only running/non-running not showing

**Comment by CaiYiLiang** [comment]
Hi @jacogr 

I think this command works. 

![image](https://user-images.githubusercontent.com/14360297/67729550-64654280-fa56-11e9-9bfb-4590b282ee20.png)

**Comment by jacogr** [comment]
Yes. It started working. I had issues around the same time with the hosted node. 

However the run/stop display button display def. does not work, but that is minor/cosmetic.

**Comment by jacogr** [comment]
Closed in #1804 - Once again, thanks an absolute million!
